generator/autogenlist.ts

```diff
@@ -56,6 +56,37 @@ const autoGenList: AutoGenConfig[] = [
         basePath: 'automation/resource-manager',
         namespace: 'Microsoft.Automation',
     },
+    {
+        basePath: 'authorization/resource-manager',
+        namespace: 'Microsoft.Authorization',
+        resourceConfig: [
+            {
+                type: 'roleAssignments',
+                scopes:ScopeType.Extension
+            },
+            {
+                type:'roleDefinitions',
+                scopes:ScopeType.Extension
+            },
+            {
+                type: 'roleAssignmentScheduleRequests',
+                scopes:ScopeType.Extension
+            },
+            {
+                type: 'roleEligibilityScheduleRequests',
+                scopes:ScopeType.Extension
+            },
+            {
+                type: 'roleManagementPolicyAssignments',
+                scopes:ScopeType.Extension
+            },
+            {
+                type: 'roleAssignmentApprovals/stages',
```

Comment:
This looks like it's being used at the tenant scope in the swagger examples: https://github.com/Azure/azure-rest-api-specs/blob/1b0ed8edd58bb7c9ade9a27430759527bd4eec8e/specification/authorization/resource-manager/Microsoft.Authorization/preview/2021-01-01-preview/examples/GetRoleAssignmentApprovals.json#L10